packages/contracts/src/RelayHub.sol

```diff
@@ -381,7 +382,14 @@ contract RelayHub is IRelayHub, Ownable {
     }
 
     function calculateCharge(uint256 gasUsed, GsnTypes.RelayData calldata relayData) public override virtual view returns (uint256) {
-        return relayData.baseRelayFee.add((gasUsed.mul(relayData.gasPrice).mul(relayData.pctRelayFee.add(100))).div(100));
+        uint256 basefee;
+        if (relayData.maxFeePerGas == relayData.maxPriorityFeePerGas) {
+            basefee = 0;
+        } else {
+            basefee = block.basefee;
+        }
+        uint256 chargeableGasPrice = Math.min(relayData.maxFeePerGas, Math.min(tx.gasprice, basefee + relayData.maxPriorityFeePerGas));
```

Comment:
math.min accepts any number of parameters and returns the lowest one...

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/min

Reply:
Alas, this is Solidity and not Nodejs, so Math.min accepts only two parameters here...

Reply:
Oh, ok... Got confused by identical look of `Math.min`